README.md

```diff
@@ -21,68 +21,169 @@ To install shelf-lib-js for development, run the following commands:
 Usage
 -----
 
-Example:
-
     // Import the library.
-    var shelfLib = require("./lib/app")("<URL where pyshelf is hosted>");
-
-    // Or, you can pass in an options object:
-    // var libOptions = {
-            logLevel: "debug",          // Sets the desired amount of logging.
-                                        // Values can be: "info", "debug", or "warning".
-                                        // Defaults to "warning".
-            "strictHostCheck": false    // Turns off request strictHostCheck.
-                                        // Defaults to true.
-        };
-    // var shelfLib = require("./lib/app")("<URL where pyshelf is hosted>", libOptions);
+    var shelfLib = require("shelf-lib")("<URL where pyshelf is hosted>");
+
+
+Options
+--------
+
+The second parameter when requiring the library is an object holding various options that can change the behavior of the library. They are detailed in this example code:
+
+    var libOptions = {
+        logLevel: "debug",          // Sets the desired amount of logging.
+                                    // Values can be: "info", "debug", or "warning".
+                                    // Defaults to "warning".
+        "strictHostCheck": false    // Turns off request strictHostCheck.
+                                    // Defaults to true.
+    };
+
+    var shelfLib = require("shelf-lib")("<URL where pyshelf is hosted>", libOptions);
```

Comment:
No. I wanted to show that there are two ways you can require the library, one with the libOptions object, and one without. But this isn't explicitly said. I will add.